spec/filters/bugs/language.rb

```diff
@@ -148,6 +148,15 @@
   fails "Magic comments in the main file do not cause bytes to be mangled by passing them through the wrong encoding" # NoMethodError: undefined method `tmp' for #<MSpecEnv:0x8aa2e>
   fails "Magic comments in the main file must be at the first line" # NoMethodError: undefined method `tmp' for #<MSpecEnv:0x8aa2e>
   fails "Magic comments in the main file must be the first token of the line" # NoMethodError: undefined method `tmp' for #<MSpecEnv:0x8aa2e>
+  fails "Magic comments in an -e argument can take Emacs style" # ArgumentError: unknown encoding name - locale
```

Comment:
I've added a few more specs in `filters/bugs` because more specs are now executed. 
For instance the following spec were previously ignored:

```rb
  platform_is_not wordsize: 32 do
    it "raises a RangeError when passed a big negative value" do
      lambda { 42.round(fixnum_min) }.should raise_error(RangeError)
    end
  end
```

```rb
  platform_is wordsize: 64 do
    describe "Array#pack with format 'L' with modifier '_'" do
      it_behaves_like :array_pack_64bit_be, 'L_'
    end

    describe "Array#pack with format 'L' with modifier '!'" do
      it_behaves_like :array_pack_64bit_be, 'L!'
    end

    describe "Array#pack with format 'l' with modifier '_'" do
      it_behaves_like :array_pack_64bit_be, 'l_'
    end

    describe "Array#pack with format 'l' with modifier '!'" do
      it_behaves_like :array_pack_64bit_be, 'l!'
    end
  end
```